services/slam/builtin/builtin.go

```diff
@@ -587,7 +395,6 @@ func NewBuiltIn(ctx context.Context, deps registry.Dependencies, config config.S
 		cancelFunc:            cancelFunc,
 		logger:                logger,
 		bufferSLAMProcessLogs: bufferSLAMProcessLogs,
-		dev:                   svcConfig.Dev,
```

Comment:
Added Kat as reviewer to confirm that this will not pose an issue. Initial plan is to push this PR as it wont break anything and then a follow up on to remove from json definition in slam repo

Reply:
https://viaminc.slack.com/archives/C036N5GJFLM/p1678392026587099?thread_ts=1677863862.260849&cid=C036N5GJFLM

Reply:
Associated PR: https://github.com/viamrobotics/slam/pull/176